README.md

```diff
@@ -483,14 +541,89 @@ that can be used to submit SQL queries. The endpoint is accessible under
 SQL statements are sent as the body of POST requests in JSON format, where the
 SQL statement is the value of the `stmt` attribute.
 
-### Read Schemas
+**QuantumLeap** offfers an API wrapping CrateDB backend so you can also perform
+multiple types of queries directly talking to QuantumLeap. The documentation of
+the API is [here](https://app.swaggerhub.com/apis/smartsdk/ngsi-tsdb/). Mind the
+versions. If you have access to your quantumleap container (e.g. it is running
+in locahost or port-forwarding to it), you can navigate its API via 
+[http://localhosts:8668/v2/ui](http://localhosts:8668/v2/ui).
+
+When to query **CrateDB** and when **QuatumLeap**?. As a rule of thumb, prefer 
+working always with **QuantumLeap** for the following reasons:
+
+- Your experience will be closer to FIWARE NGSI APIs like Orion's.
+- Your application will not be tied to CrateDB's specifics nor QuantumLeap's 
+implementation details, which could change and break your app.
+- QuantumLeap can be easily extended to other backends and your app will get 
+compatibility for free.
+- If your deployment is distributed, you won't need to expose the ports of 
+your database to the outside.
+
+If your are sure your query is not supported by **QuantumLeap**, you may have 
+to end up querying **CrateDB**, however, please open an issue in 
+[QuantumLeap's repo](https://github.com/smartsdk/ngsi-timeseries-api/issues) 
+so the team is aware.
+
+### :three: Checking Data persistence
+
+#### Using QuantumLeap API
+
+To check you are persisting `luminosity` values of entities of type `Lamp`, 
```

Comment:
For user familiarity, the **initial** persistence check should be done using a subscription query.

Checking using QuantumLeap is introducing new learning material before reinforcing old learning material. Please add the subscription check first before adding the QuantumLeap check.

Reply:
Well, with subscription query you don't check persistence, only the connection of the components. You're only sure when you see data come back hehe.  Changed this to attend your point.